config/datasets/HEAD-NECK-RADIOMICS-HN1.yaml

```
@@ -45,4 +47,6 @@ ANALYSIS:
         split_variable: {}
         impute: null
 
-RANDOM_SEED: 10
\ No newline at end of file
+RANDOM_SEED: 10
+
+VOL_LINE: 0.6803
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)